football_dictionaries/assignment_1.py

```diff
@@ -1,2 +1,3 @@
 def players_as_dictionaries(squads_list):
-    pass
+    new_squads_data = [{'number': x[0], 'position': x[1], 'name': x[2], 'date_of_birth': x[3], 'caps': x[4], 'club': x[5], 'country': x[6], 'club_country': x[7], 'year': x[8],} for x in squads_list]
```

Comment:
Hey Daniel (?),
Looks like you've created a list and added the dictionairy and for loop within the new squads data variable. I've done it a bit different. It's interesting to see that it also works like this.

And you don't have to append. Seems your solutions is shorter. is that why you have the x before the value of your key? Initialy i though the x was redundant.